runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/windmill/client/grpc/GrpcWindmillStreamFactory.java

```diff
@@ -162,7 +167,14 @@ public GetDataStream createGetDataStream(
         getDataThrottleTimer,
         jobHeader,
         streamIdGenerator,
-        streamingRpcBatchLimit);
+        streamingRpcBatchLimit,
+        sendKeyedGetDataRequests,
+        processHeartbeatResponses);
+  }
+
+  public GetDataStream createGetDataStream(
+      CloudWindmillServiceV1Alpha1Stub stub, ThrottleTimer getDataThrottleTimer) {
+    return createGetDataStream(stub, getDataThrottleTimer, false, (response) -> {});
```

Comment:
I'm not quite sure what value to put here. My thought is, since we aren't setting the processHeartbeatResponses, it doesn't really  matter (since these data streams won't be used for heartbeats).

Reply:
Fine for now, but let's remove this version since the other path (direct path related and not yet used) should handle failed heartbeats as well.